client/src/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import '../index.css';
+import { authenticationRepository, accountRepository } from '@/_services';
+import config from 'config';
+const signalR = require('@aspnet/signalr')
+
+class Dashboard extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            accountId: null,
+            currentPlan: null,
+            numberOfUsers: null,
+            maximumNumberOfUsersAllowed: null,
+            pricePerMonth: null,
+            hub: null
```

Comment:
why `hub` is part of the state, is it supposed to change?

Reply:
Good point, removed it from state.